src/components/QuranReader/TranslationView/index.tsx

```diff
@@ -15,13 +16,20 @@ import TranslationText from './TranslationText';
 type TranslationViewProps = {
   verses: Verse[];
   quranReaderStyles: QuranReaderStyles;
+  chapters: Record<string, Chapter>;
 };
 
-const TranslationView = ({ verses, quranReaderStyles }: TranslationViewProps) => (
+const TranslationView = ({ verses, quranReaderStyles, chapters }: TranslationViewProps) => (
   <div className={styles.container}>
     {verses.map((verse) => (
       <div key={verse.id}>
-        {verse.verseNumber === 1 && <ChapterHeader chapterId={String(verse.chapterId)} />}
+        {verse.verseNumber === 1 && (
+          <ChapterHeader
+            chapterId={String(verse.chapterId)}
+            nameSimple={chapters[verse.chapterId.toString()].nameSimple}
```

Comment:
Should we combine 2 `verse.chapterId.toString()` into a variable e.g. `const chapterIdString = verse.chapterId.toString()` and use it for `nameSimple` and `translatedName`?